.github/Issue_Template/Feature_Update.yaml

```diff
@@ -0,0 +1,20 @@
+name: 💡 Feature Request
+description: Have a new idea/feature for LinkFree? Please suggest!
```

Comment:
Please change the name of the project from Linkfree to Kubesimplify.

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

Reply:
Please check the latest commits